pkg/server/server_test.go

```diff
@@ -255,11 +261,14 @@ func TestResolveAuthorizationModel(t *testing.T) {
 			datastore: mockDatastore,
 			transport: transport,
 			logger:    logger,
+			typesystemResolver: typesystem.MemoizedTypesystemResolverFunc(
+				typesystem.NewTypesystemResolver(mockDatastore),
+			),
 		}
```

Comment:
same here

Reply:
See https://github.com/openfga/openfga/pull/623

Will rebase after that is merged.